client/pom.xml

```diff
@@ -11,17 +11,23 @@
     <name>tds-student-service-client</name>
     <description>The client classes for interacting with the student service</description>
 
+    <parent>
+        <groupId>org.opentestsystem.student</groupId>
+        <artifactId>tds-student-service</artifactId>
```

Comment:
I think we should rename the parent artifact to `tds-student`, that'll make it consistent with the other services (e.g. `tds-config` for TDS_ConfigService and `tds-exam` for TDS_ExamService.